### DISCUSSION TITLE
How to make a so library

### DISCUSSION THREAD

**Discussion by maccoy54** [sijms/go-ora#214] [Q&A] [answered]
Hello,

I am looking for how to make a shared library with v2. Has anyone ever tried to do this with go install or gccgo??
Best regards,

BACQ Js

**Comment by sijms** [comment]
I think you need to search how to make shared library with Golang

**Comment by sijms** [comment]
go-ora v2 has no special instruction and doesn't need cgo for build

**Comment by maccoy54** [comment]
Thank you for the answer.
I actually manage to build a program to use the module but I would like to make a shared library of this ora-go module to share it between my Docker containers. Is it possible ?

**Comment by sijms** [comment, marked as answer]
you can make shared library with Golang follow this link [go shared library](https://stackoverflow.com/questions/1757090/shared-library-in-go)
you can make this available for container follow the following [shared library for containers](https://stackoverflow.com/questions/33337457/go-shared-libraries-for-docker-containers-to-share)
every program is a separate process thus no conflict
